### DISCUSSION TITLE
Welcome to homebridge-daikin-oneplus Discussions!

### DISCUSSION THREAD

**Discussion by jeffschubert** [jeffschubert/homebridge-daikin-oneplus#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by firestorm411** [comment]
Hello Jeff: Your Daikin OnePlus plugin worked great for me for one day. I did not read the part about requesting the integration token from the Daikin One app. I am going to try this again  ##today. The reason for this note to you is with regards to compatibility. I have the newer Daikin One Touch thermostat. Do know of anyone using this thermostat with your plugin? I'm guessing it should work, given my limited success todate. Thank you for your great plugin.

**Reply by jeffschubert** [reply]
@firestorm411 Apologies for the delayed response. I don't know of anyone using it with that thermostat. But I'd be very surprised if it didn't work. The plug-in communicates with the cloud, not to the thermostat directly, so unless they significantly change the api just for that one, it should work.

**Comment by CanicheRoyal** [comment]
Great plug-in.  Worked without the token.  One feature request, and I know this is unusual, is for more than one thermostat.  We have two in our home.  I've set it up with the main floor thermostat which is the priority.  Anyway really appreciate this.  Exceeded expectations for ease of use and integration.

**Reply by CanicheRoyal** [reply]
Update:  I was happily surprised the second thermostat appeared in Home.  Great work.

**Reply by jeffschubert** [reply]
Great! I wrote it to show any devices associated with the account so I'm glad to hear both showed up.
